examples/teleportation.ipynb

```diff
@@ -0,0 +1,634 @@
+{
```

Comment:
<p>If this is a notebook focus on teleportation, it would be very helpful to have some more description on what is quantum teleportation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/qutip/qutip-notebooks/pull/108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Perhaps, I will add another notebook to go through the different measurement methods, this notebook can both serve as teleportation example (I will elaborate on that further) and serve as introduction to the run method for the circuit class ?